src/WaxSigningApi.ts

```diff
@@ -64,11 +84,18 @@ export class WaxSigningApi {
   ): Promise<ISigningResponse> {
     if (this.canAutoSign(transaction)) {
       try {
-        return await this.signViaEndpoint(
+        const startTime = Math.floor(new Date().getTime());
```

Comment:
@niksad8 I think we might also want to send a `waxjs.signing.manual.started` metric whenever we start the process so we can tell if there is a major disparity between begin and end completion of the signing process